examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -182,6 +189,8 @@ void main([List<String> args = const <String>[]]) {
       _allDemos = const JsonDecoder().convert(await driver.requestData('demoNames'));
       if (_allDemos.isEmpty)
         throw 'no demo names found';
+
+      _mediaHeight = const JsonDecoder().convert(await driver.requestData('mediaSize'))['height'];
```

Comment:
This would be a good place for a sanity check. Like `if (_mediaHeight <= 0.0) ...`

Reply:
Done.